interface/fax/fax_dispatch.php

```diff
@@ -614,203 +594,230 @@ function allCheckboxes(issel) {
+                            value='<?php echo date('Y-m-d'); ?>'
+                            data-toggle='tooltip' data-placement='top'
+                            title='<?php echo xla('yyyy-mm-dd date associated with this document'); ?>' />
+                    </div>
+                </div>
+            </div>
+            <!-- div_copy_sn Section -->
+            <div id='div_copy_sn' class='bg-secondary border rounded p-2' style='display:none;margin-top:0.5em;'>
+                <!-- Visit Date Section -->
+                <div class="form-row mt-2">
+                    <label class="col-2 col-form-label font-weight-bold"><?php echo xlt('Visit Date'); ?></label>
+                    <div class="col-10">
+                        <select name='form_copy_sn_visit' class='form-control'>
+                        </select>
+                    </div>
+                </div>
+                <!-- Comments Section -->
+                <div class="form-row mt-2">
+                    <label class="col-2 col-form-label font-weight-bold"><?php echo xlt('Comments'); ?></label>
+                    <div class="col-10">
+                        <textarea name='form_copy_sn_comments' rows='3' cols='30' class='form-control'
+                            data-toggle='tooltip' data-placement='top'
+                            title='Comments associated with this scanned note'>
+                        </textarea>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <!-- Create Patient Note Section -->
+    <div class="form-gruop row">
+        <div class="col-12 col-form-label">
+            <input type='checkbox' name='form_cb_note' value='1'
+                onclick='return divclick(this,"div_note");' />
+            <label class="font-weight-bold"><?php echo xlt('Create Patient Note'); ?></label>
+            <!-- div_note Section -->
+            <div id='div_note' class='bg-secondary border rounded p-2' style='display:none;'>
+                <!-- Type Section -->
+                <div class="form-row mt-2">
+                    <label class="col-2 col-form-label font-weight-bold"><?php echo xlt('Type'); ?></label>
+                    <div class="col-10">
+                        <?php
+                        // Added 6/2009 by BM to incorporate the patient notes into the list_options listings
+                        generate_form_field(array('data_type' => 1,'field_id' => 'note_type','list_id' => 'note_type','empty_title' => 'SKIP'), '');
+                        ?>
+                    </div>
+                </div>
+                <!-- To Section -->
+                <div class="form-row mt-2">
+                    <label class="col-2 col-form-label font-weight-bold">To</label>
+                    <div class="col-10">
+                    <select name='form_note_to' class='form-control'>
```

Comment:
When using the label class, use the "for" attribute followed by the name of the form input you are describing to allow better accessibility, for example this would be `<label for="form_note_to" class="col-2 col-form-label font-weight-bold">`. Also this one needs to have the php xlt language translation added to the label.

Reply:
Thanks for your explanation, but it seems like the label for attribute needs to follow the id, not the name.